examples/demo/demo.py

```diff
@@ -1,23 +0,0 @@
-# -------------------------------------------------------------------------------
```

Comment:
I guess we deleted this file to remove any confusion about how to interact with the examples - the examples are meant to be used via the etsdemo application. Right?

Reply:
yep, that is correct